package.json

```diff
@@ -59,11 +59,12 @@
     "node": ">=6.0.0"
   },
   "nyc": {
-    "include": [
-      "bin/templates/cordova/**",
-      "bin/lib/**",
-      "bin/create",
-      "bin/update"
+    "all": true,
+    "exclude": [
+      "tests/",
+      "bin/templates/",
+      "cordova-js-src/",
```

Comment:
I realize that this had not been included in coverage reports before, but IMHO we shouldn't be excluding this, should we? Although including it would probably hurt the coverage score :wink:

Reply:
Yeah, it would trash the score. 

Maybe we could include them in another PR. I didn't think of testing these as they are partials for `cordova-js` or partials to the end app code. Not sure what type of tests these would be.

Reply:
I think that coverage reports should include all files that are part of the production code, regardless if they are already tested or not. That being said, we don't have to change this in this PR, since these files are not tested in any platform and the topic is on my radar. And as you say, it's still a bit unclear how to test them.